segmentation/metrics/overlap_metrics.py

```diff
@@ -51,3 +52,20 @@ def mean_false_negatives(self, prediction, target):
         return mean_metric(
             prediction, target, lambda p, t: self.false_negatives_map(p, t).sum()
         )
+
+    @staticmethod
+    def false_positive_matrix(prediction, target):
+        """
+        Compute matrix of false positives between channels.
+        Target is expected to be a discrete one hot encoded label map.
+        Prediction can be discrete or continuous.
+        """
+        n_channels = target.shape[1]
+        fp_matrix = torch.zeros((n_channels, n_channels)).to(prediction.device)
```

Comment:
c'est un détail ici, car la matrice est petite, donc peu de donnée a transferer, mais autant prendre de bonne habitude,
J'ai lu quelque part que l'utilisation de .to(device) est a éviter car elle implique un transfert des données cpu to gpu.
Du coup quand c'est possible il faut initialise directement sur le gpu
https://discuss.pytorch.org/t/creating-tensors-on-gpu-directly/2714
je sais pas si il y a d'autre endroit dans le code, ou on pourait corriger, 
d=time.time(); torch.cuda.FloatTensor(1000, 1000).fill_(0)  ; print('time {}'.format(time.time()-d));                                                                                              
time 0.004454374313354492

 d=time.time(); torch.zeros(1000, 1000, device=torch.device('cuda'))  ; print('time {}'.format(time.time()-d));                                                                                     
time 0.004109859466552734
d=time.time();  torch.zeros(1000, 1000).to(device=torch.device('cuda'))  ; print('time {}'.format(time.time()-d));                                                                                                          
time 0.02329540252685547

d'un point de vue syntaxe je prefère
 torch.zeros(1000, 1000, device=torch.device('cuda'))